src/test/java/uk/gov/hmcts/cmc/claimstore/rules/CountyCourtJudgmentRuleTest.java

```diff
@@ -98,4 +101,53 @@ public void shouldThrowExceptionWhenCountyCourtJudgmentWasForAdmissionResponse()
             ).build();
         countyCourtJudgmentRule.assertCountyCourtJudgementCanBeRequested(claim, CountyCourtJudgmentType.ADMISSIONS);
     }
+
+    @Test
+    public void shouldReturnFalseWhenIsCCJDueToSettlementBreachAndClaimHasNoSettlement() {
+        Claim claim = SampleClaim.builder()
+            .withResponse(SampleResponse.validDefaults())
+            .withCountyCourtJudgmentRequestedAt(now())
+            .withCountyCourtJudgment(
+                SampleCountyCourtJudgment.builder()
+                    .paymentOption(PaymentOption.IMMEDIATELY)
+                    .build()
+            ).build();
+        assertThat(countyCourtJudgmentRule.isCCJDueToSettlementBreach(claim)).isFalse();
+    }
+
+    @Test
+    public void shouldReturnFalseWhenIsCCJDueToSettlementBreachAndClaimSettlementDateIsInTheFuture() {
+        Settlement settlement = new Settlement();
+        settlement.makeOffer(SampleOffer.builderWithPaymentIntention().build(), MadeBy.DEFENDANT);
+        settlement.accept(MadeBy.CLAIMANT);
+
+        Claim claim = SampleClaim.builder()
+            .withResponse(SampleResponse.validDefaults())
+            .withSettlement(settlement)
+            .withCountyCourtJudgmentRequestedAt(now())
+            .withCountyCourtJudgment(
+                SampleCountyCourtJudgment.builder()
+                    .paymentOption(PaymentOption.IMMEDIATELY)
+                    .build()
+            ).build();
+        assertThat(countyCourtJudgmentRule.isCCJDueToSettlementBreach(claim)).isFalse();
+    }
+
+    @Test
+    public void shouldReturnTrueWhenIsCCJDueToSettlementBreachAndClaimSettlementDateIsInTheFuture() {
```

Comment:
Review this test please - is it not settlement payment date in past?

Reply:
age old developer to command+x and command+v gone wrong.